src/modules/api/user/user.controller.ts

```diff
@@ -1,7 +1,19 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param, Post, Body } from '@nestjs/common';
 import { UserService } from './user.service';
+import { User } from 'src/schemas/user.schema';
 
-@Controller('users/:id')
+@Controller('users')
 export class UserController {
   constructor(private userService: UserService) {}
+
+  @Get()
+  getAllUsers() {
```

Comment:
the linked issue specifies only
users/:id route.
either way, we do not need a get all users anywhere right?

Reply:
+1